blazor/common/localization.md

```diff
@@ -436,9 +438,53 @@ app.Run();
 
 {% endhighlight %}
 
+{% highlight c# tabtitle=".NET 8 (~/Program.cs)" hl_lines="12 13 16 17 18 19 20 21 22 23 34" %}
+
+@using System.Globalization
+@using Microsoft.AspNetCore.Localization
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
```

Comment:
@Bhuvaneshwari-SF4208 - Why too many changes? we can mention required config alone.

Just check other code snippet examples in UG.